src/models.py

```diff
@@ -0,0 +1,104 @@
+import os
+import sys
+import logging
+import logging.config
+
+from sqlalchemy import create_engine, Column, Integer, String, Date
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+
+import config
+from helpers import create_connection, get_session
+import argparse
+import configparser
+
+logging.config.fileConfig(config.LOGGING_CONFIG)
+logger = logging.getLogger('database_models')
+
+Base = declarative_base()
+
+class Cases(Base):
+    """ Defines the data model for the table `cases`. """
+    __tablename__ = 'cases'
+    id = Column(String(100), primary_key=True, unique=True, nullable=False)
+    country_id = Column(String(100), nullable=False)
+    country = Column(String(100), unique=False, nullable=False)
+    date = Column(Date, nullable=False)
+    confirm_cases = Column(Integer, nullable=False)
+    fatal_cases = Column(Integer, nullable=False)
+
+    def __repr__(self):
+        cases_repr = "<Cases(country_id='%s', country='%s', date='%s', confirm_cases='%s', fatal_cases='%s')>"
+        return cases_repr % (self.country_id, self.country, self.date, self.confirm_cases, self.fatal_cases)
+
```

Comment:
A prediction column regarding the prediction results can be added to the 'cases' table.

Reply:
The prediction column is added.